scripts/start-helm-tests.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
```

Comment:
consider renaming this file `run-helm-tests.sh`, moving the `install_chart_and_run_tests` method from `helm-test-runner.sh` to this file, and renaming `helm-test-runner.sh` to just `helm.sh`